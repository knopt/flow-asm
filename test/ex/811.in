23
2
5.259384008175669
1.853883865265452 -1.2558382646394506 4.589269187230853 -2.7918784957081133 -3.1416598477654496 -2.9455440003030553 -0.6460349301532062 3.3960842153001103 5.807050866395613 5.05317958651365 -0.21032487742192174 7.876676693466159 -0.7627972307977515 7.281109740051413 7.700553607030578 -7.986157965934579 3.844686604525833 -0.9972170266188574 9.099731113649085 3.535669229913177 -4.132562608314531 2.659507005097053 8.705394026425331 
2.1043147526965207 -6.204267608876184 2.8764587034646745 -7.694877091594692 -3.835371129658018 -9.690893833548254 1.1084532522610964 -2.1535727125960324 -3.5632296651507023 -3.5142063444136955 1.9183294096536567 0.7781478133745967 1.256083071439944 2.333314820532559 -2.7956937530717134 -7.518636999324517 3.108148804813702 -7.32418789930038 7.738738783794755 8.416475537277034 3.452136407152807 -8.54858882270518 4.379168240158968 

4.639890943712363 4.109134632935682 
