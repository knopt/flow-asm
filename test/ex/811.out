29.684906966732264 6.746029460986515 -91.9280783556607 40.05362120088697 -28.897637378619056 -44.13332016888715 -51.08289534395247 -59.082279178490936 -98.02302850881024 -81.35809944294111 21.291495291137444 -103.32709573572123 63.39800226433395 -92.73516833086285 -77.93762811263292 104.2737987931974 -122.0160856996041 12.783939473723985 -137.54167476981954 80.57479875862083 142.08866854232917 -87.84153285935736 -136.59085167888622 
24.666889036130513 105.90036075141332 -57.60757657000501 138.2976322342549 -14.997372571404359 91.02687016794847 -86.23851551803007 52.11917148336605 89.73430527858145 90.31120485408555 -21.361491687306483 38.90993247829401 22.944615462532454 6.406450651419436 132.38130657357706 94.90583057870431 -107.25768299842474 139.5608463129403 -110.27074192013141 -17.22453349382495 -9.89001127730212 136.7410911482144 -49.90394660096652 
